packages/dbgpt-app/src/dbgpt_app/openapi/api_v1/agentic_data_api.py

```
@@ -1023,6 +1079,97 @@ async def _cancel_and_await_agent_task(task: "asyncio.Task[Any]") -> None:
+    """
+    owner_id = (user_token.user_id if user_token else None) or dialogue.user_name
+    try:
+        attachment_ctx = await prepare_react_attachments(dialogue, owner_id=owner_id)
+        if attachment_ctx is None:
+            # Legacy ``ext_info.file_path`` requests are confined to the
+            # authenticated owner's ``python_uploads/<owner>`` root; invalid
+            # input raises the same 400 and ownership failures the same
+            # non-enumerating 404 as the file_ids flow.
+            try:
+                spec = dialogue.file_input_spec()
+            except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)